Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_can_i_reduce_the_frequency_at_which_system_metrics_are_logged.md

---

> title: How can I reduce the frequency at which System Metrics are logged?

Reviewer: ```suggestion
title: How can I reduce how frequently to log system metrics?
```

toc_hide: true
type: docs
tags:
  - metrics
  - runs
---

> Adjust the sampling interval (default is 10s):

Reviewer: ```suggestion
To configure the frequency to log [system metrics]({{< relref "/guides/models/app/settings-page/system-metrics.md" >}}), set `_stats_sampling_interval` to a number of seconds (default: `10.0`), expressed as a float:
```
Again, no mention of this setting in the public docs...
